.github/scripts/artifactory.spec

```diff
@@ -0,0 +1,23 @@
+{
+"files": [
+  {
+    "aql": {
+      "items.find": {
+        "repo": {"$eq":"dart.yellow.private"},
```

Comment:
We need to change the name to the new one

Reply:
Done